File: Libraries/Internal/ACVPCore/Models/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ACVPCore.Models
{
    public class Person
    {
        public long ID { get; set; }
        public string Name { get; set; }
        public Organization Organization { get; set; }
        public List<PersonPhone> PhoneNumbers { get; set; }
        public List<string> EmailAddresses { get; set; }
        public List<PersonNote> Notes { get; set; }
    }
}

File: Web.Admin/ClientApp/src/app/models/person/Person.ts
import { Organization } from '../organization/Organization';
import { PersonPhone } from './PersonPhone';
import { PersonNote } from './PersonNote';

export class Person {

  id: number;
  name: string;
  organization: Organization;
  phoneNumbers: PersonPhone[];
  emailAddresses: string[];
  notes: PersonNote[];

  public OperatingEnvironment(id: number,
    name: string,
    organization: Organization,
    phoneNumbers: PersonPhone[],
    emailAddresses: string[],
    notes: PersonNote[]) {
    this.id = id;
    this.name = name;
    this.organization = organization;
    this.phoneNumbers = phoneNumbers;
    this.emailAddresses = emailAddresses;
    this.notes = notes;
  };

}
